### ISSUE TITLE
make distinction between range and uncertainty clearer and have subtypes

### ISSUE THREAD

**Issue by kjappelbaum** [cheminfo/cheminfo-json-schema#20]
- have separate types for value and range
- make clear that range 
- how do you save the uncertainty

**Comment by kjappelbaum** [comment]
do we have something like 

```ts
export interface QuantityWithUnit {
  value: number;
  unitText: string; 
  unitCode: string|URL; 
}
```

**Comment by kjappelbaum** [comment]
closes as duplicate of #22
